x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -463,7 +474,7 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
             onFilterIndexPatterns={filterIndexPatterns}
           />
 
-          {listType !== ExceptionListTypeEnum.ENDPOINT && (
+          {listType !== ExceptionListTypeEnum.ENDPOINT && !sharedListToAddTo?.length && (
```

Comment:
No, I mean the `!sharedListToAddTo?.length` still, because I want to hide the Add-To-Rule/List section in case the `sharedListToAddTo` has a value 

``!!sharedListToaddTo?.length `` this expression will return true if the `sharedListToAddTo` array has values=> because `!!` is a casting operator to convert from number to boolean